IDE/Espressif/ESP-IDF/libs/component.mk

```diff
@@ -13,5 +13,4 @@ CFLAGS +=-DWOLFSSL_USER_SETTINGS
 
 COMPONENT_OBJEXCLUDE := wolfcrypt/src/aes_asm.o
 COMPONENT_OBJEXCLUDE += wolfcrypt/src/evp.o
-COMPONENT_OBJEXCLUDE += wolfcrypt/src/misc.o
```

Comment:
I think that your settings would have NO_INLINE definition. Current user_settings.h for ESP-IDF port doesn't have NO_INLINE definition as default. The misc.c is included when expanding to INLINE. Therefore, the file excludes from target file in makefile.
Because the user_settings.h is good for default, I think that we don't need to remove this.

Reply:
Thanks, there is indeed NO_INLINE, I will do some testing and feedback here.